essay4.md

```diff
@@ -72,19 +72,8 @@ Lastly, [frame_test.py](https://github.com/commaai/openpilot/blob/master/system/
     * msmb_ispif.h - Short file. Makes sure proper bits being addressed to. 
 
 ### Snapshot
-* methods
-  * extract_image() - get the image in YUV and converts it to RGB for processing by calling yuv_to_rgb()
-  * get_snapshots() - gets an image from the rear of the vehicle and the front
-  * jpeg_write() - creats an image memory and saves it
-  * snapshot() - works like the main function for snapshots calling all of the other processes and returning the images from get_snapshots()
-  * yuv_to_rgb() - converts picture from YUV to RGB
-
-### init
-* nothing in here?
+#### methods
+There are many methods for snapshots to be created and processed for openpilot. The first method is extract_image().It gets the image in YUV and converts it to RGB for processing by calling yuv_to_rgb. The yuv_to_rgb method converts picture from YUV to RGB. YUV images use less bandwidth that RGB images, which will allow the image to be transported faster. The method get_snapshots recieves an image from the rear and front of the vehicle to use for processing and eventually decision making. To create an image memory and save it, jpeg_write is used. THe snapshot method works like the main function for snapshots calling all of the other processes and returning the images from get_snapshots.
```

Comment:
There are a couple of small typos in this line: a space after the second sentence, change the words "bandwidth that RGB" to "bandwidth than RGB", and there is a capital H in "THe" in the last sentence

Reply:
fixed